src/libraries/Microsoft.PowerFx.Core/Syntax/Nodes/SelfNode.cs

```diff
@@ -9,34 +9,37 @@
 
 namespace Microsoft.PowerFx.Core.Syntax.Nodes
 {
-    internal sealed class SelfNode : NameNode
+    public sealed class SelfNode : NameNode
```

Comment:
>SelfNode [](http://example.com/codeflow?start=24&length=8)

Note: Self and Parent should not be special node types. Before GA, we should remove these nodes, and just make them special reserved identifiers in Canvas.

Reply:
added #257

Reply:
Independent followup, needed before GA after this is checked in.